<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recovery_form</class>
 <widget class="QWidget" name="Recovery_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Восстановить</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Файл с настройками в формате *.txt:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Каталог с изображениями в формате *.bmp или каталог со слоями:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="TxtFilePath"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="OkButton">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="FileButton">
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="CatalogPath"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="CatalogButton">
     <property name="text">
      <string>Выбрать каталог</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="CanctlButton">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Тип панорамы:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="panorama_2D">
        <property name="text">
         <string>2D</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="Panorama_3D">
        <property name="text">
         <string>3D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Размер панорамы:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QSpinBox" name="VertycalValue">
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSpinBox" name="HorizontalValue">
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>    X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
